### DISCUSSION TITLE
product Variation limite !!!!

### DISCUSSION THREAD

**Discussion by opestro** [scottyzen/woonuxt#126] [General]
hello , i have product with 21 variant, but when retreving product data i get only 10
how can i fiw this ? or change the limite params !
![image](https://github.com/scottyzen/woonuxt/assets/96887412/cde6c5fd-8010-4dfa-9419-a04f0b8facca)

**Comment by scottyzen** [comment]
Hi @opestro. I just pushed a change that should fix this.
